From: Kir Kolyshkin <kolyshkin@gmail.com>
Date: Thu, 5 Jul 2018 04:07:45 -0700
Subject: [PATCH] Compile fix

Go 1.11beta1 (rightfully) complains:

> 15:38:37 daemon/cluster/controllers/plugin/controller.go:183:
> Entry.Debugf format %#T has unrecognized flag #

This debug print was added by commit 72c3bcf2a5.

Signed-off-by: Kir Kolyshkin <kolyshkin@gmail.com>
Signed-off-by: Sebastiaan van Stijn <github@gone.nl>
Origin: upstream, https://github.com/moby/moby/commit/a9a1365
---
 daemon/cluster/controllers/plugin/controller.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/engine/daemon/cluster/controllers/plugin/controller.go b/engine/daemon/cluster/controllers/plugin/controller.go
index 6d7606aa8435..371f918d3996 100644
--- a/engine/daemon/cluster/controllers/plugin/controller.go
+++ b/engine/daemon/cluster/controllers/plugin/controller.go
@@ -180,7 +180,7 @@ func (p *Controller) Wait(ctx context.Context) error {
 		case <-ctx.Done():
 			return ctx.Err()
 		case e := <-events:
-			p.logger.Debugf("got event %#T", e)
+			p.logger.Debugf("got event %T", e)
 
 			switch e.(type) {
 			case plugin.EventEnable:
